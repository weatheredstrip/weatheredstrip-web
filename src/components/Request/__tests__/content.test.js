import React from 'react'
import { shallow } from 'enzyme'
import Content from '../content.react'
import 'jest-dom/extend-expect'
import testData from './test_data.json'

// Full content is not tested yet, because timestamp is dependent on current time.
it('Content renders properly when empty', () => {
  const component = shallow(<Content />)
  expect(component).toMatchSnapshot()
})

it('content renders properly with data', () => {
  const content = shallow(<Content data={testData} />)
  expect(content).toMatchSnapshot()
})
