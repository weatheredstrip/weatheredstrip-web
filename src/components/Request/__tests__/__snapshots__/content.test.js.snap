// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders properly when empty 1`] = `ShallowWrapper {}`;

exports[`content renders properly with data 1`] = `ShallowWrapper {}`;
